src/exec.js

```diff
@@ -1,42 +1,68 @@
 const path = require('path');
 const { exec, spawn } = require('child_process');
 
+module.exports = (cmd, params) =>
+  new Promise((resolve, reject) => {
+    const nodeModuleFolder = path.join(__dirname, '..', 'node_modules');
+    const binFolder = path.join(nodeModuleFolder, '.bin');
+    const pathArr = [process.env.PATH || '', binFolder].filter(x => !!x);
```

Comment:
considering that path environment variable might be either `PATH` or `Path`, shouldn't we read it as `process.env.PATH || process.env.Path || ''` ? Also the `|| ''` seems a bit redundant since there is a truthy filter afterwards applied on the array.

Reply:
I had the same question actually, but js, when was reading the key, didn't care about the capital or lower case. for any case like `Path`, `path` `pATH` result was the same. so I let it be as it is

Reply:
I think they convert it to a lower letter when you are reading it. but since we copy it to a new object. then we will end up with two key instead of one

![image](https://user-images.githubusercontent.com/8991783/106637154-512c1580-65bd-11eb-9e1f-8fba35c15a6c.png)

![image](https://user-images.githubusercontent.com/8991783/106637212-6012c800-65bd-11eb-9a2c-732e1021b22e.png)